contributors/guide/contributor-cheatsheet.md

```diff
@@ -15,7 +15,7 @@ A list of common resources when contributing to Kubernetes.
 ## Workflow
 
 - [Gubernator Dashboard - k8s.reviews](https://k8s-gubernator.appspot.com/pr)
-- [Submit Queue](https://submit-queue.k8s.io)
+- [Tide](https://prow.k8s.io/tide)
```

Comment:
Please also remove reference to the cherrypick queue below.

Reply:
Done.